Changes for new Twitter limit of 280 characters.

wcz-txp opened the pull request:
Tested with Textpattern 4.6.2 and PHP 7.0.27

wcz-txp committed:
Changes for new Twitter limit of 280 characters. Tested with Textpattern 4.6.2 and PHP 7.0.27

drmonkeyninja commented:
Thanks for this. I've merged it in and added an upgrade to the `tweet` column on install for anyone who's used the plugin before the increase in tweet length.

wcz-txp commented:
Thanks, I updated my database by hand, but that way it's more convenient for all :-)



-----Original Message-----
From: Andy Carter <notifications@github.com>
To: drmonkeyninja/TXP-Tweet <TXP-Tweet@noreply.github.com>
Cc: Andrij <whocarez@web.de>, Author <author@noreply.github.com>
Sent: пн, 05 февр. 2018 23:37
Subject: Re: [drmonkeyninja/TXP-Tweet] Changes for new Twitter limit of 280 characters. (#22)

Thanks for this. I've merged it in and added an upgrade to the `tweet` column on install for anyone who's used the plugin before the increase in tweet length.

-- 
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/drmonkeyninja/TXP-Tweet/pull/22#issuecomment-363229624
